### ISSUE TITLE
Create separate openshift projects for UAT and Dev environments

### ISSUE THREAD

**Issue by MenaMachado** [CDH-Studio/I-Talent#343]
For Upskill, in openshift, there is both a set of dev and uat apps - those should be separated into different openshift projects.

1.  Create a YAML configuration for the prod project. 	

a brief instruction for that is here https://wiki.ised-isde.canada.ca/display/RDADC/OpenShift+Developer+FAQ#OpenShiftDeveloperFAQ-HowdoIcreateatemplateofmyproject?
	
2. Test the YAML out in a lower environment. I suggest they use Upskill dev for that as it would satisfy the first bullet
		
instructions for populating an openshift project are here https://wiki.ised-isde.canada.ca/display/RDADC/OpenShift+Developer+FAQ#OpenShiftDeveloperFAQ-HowdoIcreatemyprojectfromatemplate

Cloud Team will 
- Convert the s2i builds to jenkins
- push the prod image to prod
- create and populate the prod project with the yaml your team created

**Comment by BenJeau** [comment]
Template file is now done. Will just be waiting on the new backend to be merged into uat, so we only have one template file instead of two. And on this issue, which will added new pods in the template #447

**Comment by BenJeau** [comment]
https://github.com/CDH-Studio/UpSkill/wiki/OpenShift I wrote some docs which actually works for creating a template

**Comment by BenJeau** [comment]
Both openshift templates are now merged into our repos
